test/rpc/test_snapshots.py

```diff
@@ -17,7 +17,7 @@
             updated_at='updated_at',
         )
     }}
-    select 1 as id, '2019-10-31 23:59:40' as updated_at
+    select 1 as id, '2019-10-31 23:59:40'::timestamp  as updated_at
```

Comment:
I'm not opposed to casting this to timestamp here, but you'll want to do it in a way that's database-agnostic:
```suggestion
    select 1 as id, cast('2019-10-31 23:59:40' as timestamp)  as updated_at
```

Reply:
Great suggestion, wasn't sure that cast as was database agnostic (+ the test file was only for postgres if I'm not mistaken).

Reply:
you're right, I was overcautious — still, in case we want to run tests elsewhere, it doesn't hurt!